clonelists/Sony - PlayStation 2 (Redump).json

```diff
@@ -4174,20 +4198,6 @@
 				{"searchTerm": "Metal Saga - Sajin no Kusari"}
 			]
 		},
-		{
-			"group": "Metal Slug",
-			"compilations": [
-				{"searchTerm": "Metal Slug Anthology"},
-				{"searchTerm": "Metal Slug Complete"}
-			]
-		},
-		{
-			"group": "Metal Slug 2",
-			"compilations": [
-				{"searchTerm": "Metal Slug Anthology"},
-				{"searchTerm": "Metal Slug Complete"}
-			]
-		},

```

Comment:
Don't remove the groups if the individual titles don't exist -- this is how compilations are done now. It helps the compilation selection algorithm to know what titles are in the compilation. Same for `Metal Slug X` further down.